newapi/ooniapi/measurements.py

```diff
@@ -270,7 +323,31 @@ def _fetch_measurement_body_on_disk(report_id, input: str) -> Optional[bytes]:
     return ujson.dumps(body).encode()
 
 
-def _fetch_autoclaved_measurement_body(report_id: str, input) -> bytes:
+def _fetch_measurement_body_on_disk_by_msmt_uid(msmt_uid: str) -> Optional[bytes]:
+    """Fetch raw POST from disk, extract msmt
+    This is used only for msmts that have been processed by the fastpath
+    but are not uploaded to S3 yet.
+    YAML msmts not supported: requires implementing normalization here
+    """
+    assert msmt_uid.startswith("20")
+    tstamp, cc, testname, hash_ = msmt_uid.split("_")
+    hour = tstamp[:10]
+    int(hour)
+    spooldir = Path("/var/lib/ooniapi/measurements/incoming/")
```

Comment:
Should this path perhaps be defined in the config file or at least in a top level const variable?

Reply:
It's used only in that place. If we'll have to support different paths in future we can change it when needed.